src/postcss.config.ts

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Untested... we should have even a simple validation check for something like this.